data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report5_row_suppression_icd.py

```diff
@@ -26,11 +27,13 @@
 # + tags=["parameters"]
 project_id = ""
 deid_cdr=""
-run_as = ""
-# -
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns = ['query', 'result']) 
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.